db/schema.rb

```diff
@@ -418,6 +418,46 @@
     t.index ["table_name", "count"], name: "index_table_record_counts_on_table_name_and_count"
   end
 
+  create_table "token_collections", force: :cascade do |t|
+    t.string "standard"
+    t.string "name"
+    t.text "description"
+    t.integer "creator_id"
+    t.string "icon_url"
+    t.integer "items_count"
+    t.integer "holders_count"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
+
+  create_table "token_items", force: :cascade do |t|
+    t.integer "collection_id"
+    t.string "token_id"
+    t.string "name"
+    t.string "icon_url"
+    t.integer "owner_id"
+    t.string "metadata_url"
+    t.integer "cell_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["cell_id"], name: "index_token_items_on_cell_id"
+    t.index ["collection_id", "token_id"], name: "index_token_items_on_collection_id_and_token_id", unique: true
+    t.index ["owner_id"], name: "index_token_items_on_owner_id"
+  end
+
+  create_table "token_transfers", force: :cascade do |t|
+    t.integer "item_id"
+    t.integer "from_id"
+    t.integer "to_id"
+    t.integer "transaction_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["from_id"], name: "index_token_transfers_on_from_id"
+    t.index ["item_id"], name: "index_token_transfers_on_item_id"
+    t.index ["to_id"], name: "index_token_transfers_on_to_id"
+    t.index ["transaction_id"], name: "index_token_transfers_on_transaction_id"
+  end
+
```

Comment:
These codes are not related to this issue

Reply:
+1